scripts/build/builders/nrf.py

```diff
@@ -133,22 +140,11 @@ def generate(self):
                     raise Exception(
                         "Directory %s not writable. NRFConnect builds require updates to this directory." % nrfconnect_sdk)
 
-                # validate the the ZEPHYR_BASE is up to date (generally the case in docker images)
```

Comment:
is this ok to remove?

It was here because other build scripts do a full update where as this script does not.

Reply:
My bad. Reverted this change. Thanks for catching!